src/front/scripts/index.js

```diff
@@ -352,46 +352,76 @@ $(document).ready(function () {
       $(this).next('tr[data-pl-detail-row]').toggle();
   });
 
-  // Define the tour!
-  /* jshint ignore:start */
-  var tour = {
-      id: 'hopscotch-example',
-      steps: [{
-              title: 'Hopscotch Tour Example',
-              content: 'This is an example of a tour.',
-              target: 'tour',
-              placement: 'bottom'
-          },
-          {
-              title: 'Purpose',
-              content: 'Example of tour.',
-              target: 'description',
-              placement: 'bottom'
-          }, {
-              title: 'Navigation',
-              content: 'Foundation, Components, Patterns describe how best to use the CBP Theme.',
-              target: 'sidebar',
-              placement: 'right'
-          }
-
-      ],
-      showPrevButton: true
-  };
-  var calloutMgr = hopscotch.getCalloutManager();
-  
-  if (document.URL.indexOf('tours.html') > -1) {
-      // Start the tour!
-      hopscotch.startTour(tour);
-
-      calloutMgr.createCallout({
+     // Hopscotch 
```

Comment:
On load the Callout appears in the wrong place:

<img width="707" alt="tours" src="https://user-images.githubusercontent.com/1034574/60726122-ced2ff80-9f08-11e9-92b6-94abed021ff7.png">